src/builder/builder.go

```diff
@@ -200,15 +200,38 @@ func (b *Builder) UpdateRepo(ver string, allbundles bool) {
 				os.Exit(1)
 			}
 			for _, file := range files {
-				helpers.CopyFile(bundles+"/"+file.Name(), clrbundles+file.Name())
+				err := helpers.CopyFile(bundles+"/"+file.Name(), clrbundles+file.Name(), true)
+				if err != nil {
+					helpers.PrintError(err)
+					os.Exit(1)
+				}
 			}
 		} else {
 			// Install only a minimal set of bundles
 			fmt.Println("Adding os-core, os-core-update, kernel-native, bootloader to mix-bundles...")
-			helpers.CopyFile(bundles+"/os-core", clrbundles+"os-core")
-			helpers.CopyFile(bundles+"/os-core-update", clrbundles+"os-core-update")
-			helpers.CopyFile(bundles+"/kernel-native", clrbundles+"kernel-native")
-			helpers.CopyFile(bundles+"/bootloader", clrbundles+"bootloader")
+			err := helpers.CopyFile(bundles+"/os-core", clrbundles+"os-core", true)
+			if err != nil {
+				helpers.PrintError(err)
+				os.Exit(1)
+			}
+
+			err = helpers.CopyFile(bundles+"/os-core-update", clrbundles+"os-core-update", true)
+			if err != nil {
+				helpers.PrintError(err)
+				os.Exit(1)
+			}
+
+			err = helpers.CopyFile(bundles+"/kernel-native", clrbundles+"kernel-native", true)
+			if err != nil {
+				helpers.PrintError(err)
+				os.Exit(1)
+			}
+
+			err = helpers.CopyFile(bundles+"/bootloader", clrbundles+"bootloader", true)
+			if err != nil {
+				helpers.PrintError(err)
+				os.Exit(1)
+			}
```

Comment:
One of my absolute least favorite things about Go's "return an error as one of your return values" paradigm is that it results in code constantly being peppered with `if err != nill {  // Print error & exit }` blocks, just like this, where 4 lines become 16. You're not doing anything wrong here -- it's "how Go works" -- but it makes code a lot a lot harder to read IMHO.

Reply:
As a separate issue, the actual formatting of these (and many others) could/should be changed to be more idiomatic, using Go's initializer statement for `if` statements. Something like:

```go
if err = helpers.CopyFile(bundles+"/bootloader", clrbundles+"bootloader", true); err != nil {
    helpers.PrintError(err)
    os.Exit(1)
}
````

Reply:
Yeah, it was a bit painful to change that, it looked very neat before. I couldn't find any nicer way to handle the errors, so i guess we have to live with that. As for the if initializer, I still look it quite ugly, but I guess that's the Go way.... I'll update it.

Reply:
I think this is ok for now... if everyone agrees on a consistent implementation let's propagate it throughout the rest of the codebase in another PR.